releases/release-1.22/release-notes/maps/pr-102467-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 102467
+releasenote:
+  text: Fix nulls are handles of array's and objects in json patches.
```

Comment:
This is ambiguous. Can we find more information on it?

Reply:
I don't think it would be a user-facing change. Please let me know if I could removed it.

Reply:
This one might be a user-facing change. Let me try to rephrase it.

Reply:
Updated